### PR TITLE
Extracted some management types to terracotta-management module

### PR DESCRIPTION
**Pull request by mathieucarbou**
@anthonydahanne, @lorban : here is the PR for extracting types

### CODE REVIEW

File: management/src/main/java/org/ehcache/management/QueryBuilder.java [hunk 1]
```diff
@@ -15,9 +15,9 @@
  */
 package org.ehcache.management;
 
+import org.terracotta.management.context.Context;
+
 import java.util.Collection;
-import java.util.List;
-import java.util.Map;
```
Comment (anthonydahanne) [inline management/src/main/java/org/ehcache/management/QueryBuilder.java]: hum... that was not used before ? (unused import ?)

File: management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java [hunk 2]
```diff
@@ -24,13 +23,12 @@
...
 import org.terracotta.management.capabilities.descriptors.Descriptor;
+import org.terracotta.management.context.Context;
 import org.terracotta.management.stats.Statistic;
 
-import java.util.ArrayList;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
```
Comment (anthonydahanne) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java]: those types were not used ?

Comment (anthonydahanne) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java]: apparently not (just checked)

File: management/src/main/java/org/ehcache/management/QueryBuilder.java (hunk 1 above)
Comment (anthonydahanne) [inline management/src/main/java/org/ehcache/management/QueryBuilder.java]: apparently not (just checked)

**Comment by anthonydahanne** [comment]
LGTM , thanks for fixing the unused imports

**Comment by cschanck** [comment]
At Anthony's request...

**Pull request merged**